Added item component

makarman opened the pull request:
Added Item compomemt
Added Assignment subcomponent
Added Literature subcomponent
Added Answer subcomponent
Added Timeleft subcomponent

## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVPYTP-51)


## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

redmyt commented on eventually/static/src/containers/item/answer.js:
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
name of files with components should start with the uppercase letter.

skramar replied:
Components start with Uppercase, files camelCase

redmyt replied:
I found that advice in following articles: (https://daveceddia.com/react-project-structure/, https://survivejs.com/react/advanced-techniques/structuring-react-projects/) and also WebStror IDE notifies me about it =)

makarman replied:
Okey I'll use uppercase

redmyt commented on eventually/static/src/containers/item/answer.js:
```diff
@@ -0,0 +1,46 @@
+import RaisedButton from 'material-ui/RaisedButton';
+import {sendAnswer} from './itemService.js'
+
+const style = {
```
please, use 4 spaces indentation in your files =)

makarman replied:
done

redmyt commented on eventually/static/src/containers/item/answer.js:
```diff
@@ -0,0 +1,46 @@
+  };
+};
+
+export default withRouter(Answer)
```
is there necessary to use `withRouter` export?
Add new line =)

makarman replied:
done

redmyt commented on eventually/static/src/containers/item/assignment.js:
```diff
@@ -0,0 +1,27 @@
+  };
+};
+
+export default Assignment
```
new line

makarman replied:
done

redmyt commented on eventually/static/src/containers/item/item.js:
```diff
@@ -0,0 +1,57 @@
+  }
+
+  render() {
+      console.log(this.state['literatureList'])
```
forgot it =)

makarman replied:
done

redmyt commented on eventually/static/src/containers/item/literature.js:
```diff
@@ -0,0 +1,32 @@
+  };
+};
+
+export default withRouter(Literature)
```
new line

makarman replied:
done

redmyt commented on eventually/static/src/containers/item/itemService.js:
```diff
@@ -0,0 +1,19 @@
+import axios from "axios"
+
+const appPath = '/api/v1/user/'
```
are all of this path necessary?

makarman replied:
no) I'll remove it

lhalam commented on eventually/static/src/containers/home.js:
```diff
@@ -7,7 +7,7 @@ export default class Home extends React.Component {
         <div>
           <h1>Hello user!!!!</h1>
-
+          
```
remove empty line

makarman replied:
done

makarman committed:
Added item component

Added Item compomemt
Added Assignment subcomponent
Added Literature subcomponent
Added Answer subcomponent
Added Timeleft subcomponent
